spec/controllers/api/v1/books_controller_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+describe Api::V1::BooksController, type: :controller do
+  include_context 'Authenticated User'
+  describe 'GET #index' do
+    context 'When fetching all the books' do
+      let!(:books) { create_list(:book, 3) }
+
+      before do
+        get :index
```

Comment:
In the project, we put this `get` inside a subject, in other to make it  the thing that is being tested, like this:

```
subject(:http_request) { get :index }
```

Reply:
done!